src/main/java/org/axolotlik/axolotlikcosmocats/domain/Product.java

```diff
@@ -0,0 +1,17 @@
+package org.axolotlik.axolotlikcosmocats.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
```

Comment:
I would prefer you use @Value on the domain classes, as far as you can make sth immutable you should do so